phpunit/tests/blocks/renderHookedBlocks.php

```diff
@@ -1,168 +0,0 @@
-<?php
-
-/**
- * Tests for hooked blocks rendering.
- *
- * @package WordPress
- * @subpackage Blocks
- * @since 6.4.0
- *
- * @group blocks
- */
-class Tests_Blocks_RenderHookedBlocks extends WP_UnitTestCase {
```

Comment:
This class is testing the code that's bridging the gaps between WP core and Gutenberg. We can remove it unless @ockham plans to apply any changes in the compat layer.

Reply:
If I'm not wrong, the test hasn't been touched since 6.4 which means, that if we want to make any changes now, we'll have to do add a file to `6.5` folder and modify this file. It seems actually better to remove it that way, it clarifies for the implementor what part of the test need to be backported to Core for the "new" changes.

I can restore it though, just wanted to explain my reasoning here.

Reply:
Sounds like a good reasoning 👍🏻